### PR TITLE
Fix discussion exceptions

### PR DESCRIPTION
**Pull request by regiskuckaertz**
## What does this change?

Exceptions thrown by the discussions loader cannibalise the console when run locally.
## What is the value of this and can you measure success?

No more unsightly exceptions (but still reported by raven).

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/discussion/loader.js [hunk 1]
```diff
@@ -73,7 +73,9 @@ Loader.prototype.initTopComments = function() {
                 this.setState('has-top-comments');
             }
         }.bind(this)
-    );
+    ).catch(function () {
+        /* ignore */
```
Comment (jamesgorrie) [inline static/src/javascripts/projects/common/modules/discussion/loader.js]: Isn't that a little dangerous?

Comment (gtrufitt) [inline static/src/javascripts/projects/common/modules/discussion/loader.js]: Can we send them to sentry so if there are prod errors they get logged nicely?

Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/discussion/loader.js]: Yep I'll improve that now.

**Comment by jamesgorrie** [comment]
Nice 👍

**Pull request merged**